.gitignore

```diff
@@ -1,5 +1,6 @@
 strest-grpc*
 release/
+vendor/
```

Comment:
I think the need for a `vendor` directory was removed in #58, when we switched this repo to go modules. Is there something in your local env that's producing this directory? I don't see it when I run `GO111MODULE=on go run main.go` in this repo.

Reply:
I ran `go mod vendor` and it created this directory. I also don't really know what i'm doing with `go mod` so maybe it wasn't necessary...

Reply:
My understanding is that you only need the vendor directory if you're not running with go modules enabled. If you enable go modules, then that should handle fetching the dependencies automatically and storing them somewhere outside of your source tree. But yeah, idk